loleaflet/src/layer/tile/TileLayer.js

```diff
@@ -1979,7 +1979,6 @@ L.TileLayer = L.GridLayer.extend({
 					center = center.subtract(this._map.getSize().divideBy(2));
 					center.x = Math.round(center.x < 0 ? 0 : center.x);
 					center.y = Math.round(center.y < 0 ? 0 : center.y);
-					this._map.fire('scrollto', {x: center.x, y: center.y});
```

Comment:
It seems all the above code just computes the new center but does nothing with it. Is it possible to adjust the scroll to that new position in the scrollbar or just set the map's center to that so that everything automatically change to that location ?

Reply:
By the way, later i saw that i shouldn't update that part. Instead, i used map.wholeColumnSelected and map.wholeRowSelected variables. FYI, thanks :)